### DISCUSSION TITLE
Add stun/guard bar to Ikemen's default lifebars.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1446] [Feature Request] [answered]
### Additional context

Since Ikemen's default 240p/720p screenpacks are supposed to be used as reference for exclusive features would be nice to have stun and guard bar there as well.

**Comment by potsmugen** [comment]
Agreed. I think most new features should be exhibited by default. If I recall correctly, the plan is to use an original screenpack down the line, so maybe it'll be in there.

**Comment by potsmugen** [comment, marked as answer]
Already implemented in the new lifebars.
